library/dokku_letsencrypt.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# from ansible.module_utils.basic import *
+from ansible.module_utils.basic import AnsibleModule
+import subprocess
+
+DOCUMENTATION = """
+---
+module: dokku_letsencrypt
+short_description: Enable or disable the letsencrypt plugin for a dokku app
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  state:
+    description:
+      - The state of the letsencrypt plugin
+    required: False
+    default: present
+    choices: [ "present", "absent" ]
+    aliases: []
+author: Gavin Ballard
+requirements:
+  - the `dokku-letsencrypt` plugin
+"""
+
+EXAMPLES = """
+- name: Enable the letsencrypt plugin
+  dokku_letsencrypt:
+    app: hello-world
+
+- name: Disable the letsencrypt plugin
+  dokku_letsencrypt:
+    app: hello-world
+    state: absent
+"""
+
+
+def force_list(var):
+    if isinstance(var, list):
+        return var
+    return list(var)
+
+
+def subprocess_check_output(command, split="\n"):
+    error = None
+    output = []
+    try:
+        output = subprocess.check_output(command, shell=True)
+        if isinstance(output, bytes):
+            output = output.decode("utf-8")
+        output = str(output).rstrip("\n")
+        if split is None:
+            return output, error
+
+        output = output.split(split)
+        output = force_list(filter(None, output))
+        output = [o.strip() for o in output]
+    except subprocess.CalledProcessError as e:
+        error = str(e)
+    return output, error
+
+
+def dokku_letsencrypt(data):
+    command = "dokku --quiet letsencrypt:list | awk '{{print $1}}'"
```

Comment:
As the `dokku-letsencrypt` plugin does not (as far as I'm aware) provide a way to retrieve the current status of Let's Encrypt for a specific app, using `awk` and checking if the target app name is in the list of currently enabled apps was the only method I really had to do this.